### PR TITLE
RELATED: RAIL-4678 fix dropzone placement

### PR DESCRIPTION
**Pull request by MilanLempera**
- remove row wrappers in edit mode
- split DashboardLayoutGridRow components for view and edit mode
- rework height resizer rendering and  height calculation
- fix useDashboardDrag collected props type

JIRA: RAIL-4678

---

Supported PR commands:

| Command                                               | Description                                                |
| ----------------------------------------------------- | ---------------------------------------------------------- |
| `ok to test`                                          | Re-run standard checks                                     |
| `extended check sonar`                                | SonarQube tests                                            |
| `extended check plugins`                              | Dashboard plugins tests                                    |
| `extended test - backstop`                            | BackstopJS tests                                           |
| **E2E Cypress tests commands - TIGER**                |                                                            |
| `extended test - tiger-cypress - isolated <testName>` | Run isolated tests running against recorded Tiger backend. |
| `extended test - tiger-cypress - record <testName>`   | Create a new recording for isolated Tiger tests.           |
| **E2E Cypress tests commands - BEAR**                 |                                                            |
| `extended test - cypress - isolated <testName>`       | Run isolated tests running against recorded Bear backend.  |
| `extended test - cypress - record <testName>`         | Create a new recording for isolated Bear tests.            |

`<testName>` in cypress commands is used to filter specfiles. Example, to run record with BEAR backend

-   Against `dashboard.spec.ts` and `drilling.spec.ts`, execute command `extended test - cypress - record dashboard,drilling`
-   Against all specfiles, execute command `extended test - cypress - record` or `extended test - cypress - record *`

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `extended test - backstop` passes
-   [ ] `extended test - tiger-cypress - record` to record new mapping files (Tiger BE)
-   [ ] `extended test - cypress - record` to record new mapping files (Bear BE)
-   [ ] `extended test - tiger-cypress - isolated` passes
-   [ ] `extended test - cypress - isolated` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/11216/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/13330/ : FAILURE in 5m 44s

**Comment by MilanLempera** [comment]
extended test - tiger-cypress - isolated

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/11220/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/13332/ : FAILURE in 13m 41s

**Comment by yenkins** [comment]
# ✨ Cypress test results ✨

## 😢 Some tests are failing. Consider fixing them.

|Status|Name|Tests|Failures|Time|
|:--:|:--|:--:|:--:|--:|
|❌|**[🎥 dashboardTiger.spec.ts](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-docker/17//artifact/libs/sdk-ui-tests-e2e/cypress/videos/dashboardTiger.spec.ts.mp4)**|9(4 skipped)|<span style="color: red">5</span>|158.5910s|

You can find full report [here](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-docker/17/).

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher/65/ : FAILURE in 16m 41s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/11221/ : SUCCESS in 11s

**Comment by MilanLempera** [comment]
extended test - cypress - isolated

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/11222/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/13333/ : SUCCESS in 22m 01s

**Comment by yenkins** [comment]
# ✨ Cypress test results ✨

## 😢 Some tests are failing. Consider fixing them.

|Status|Name|Tests|Failures|Time|
|:--:|:--|:--:|:--:|--:|
|✅|**dashboardDnD.spec.ts**|7|-|95.3460s|
|✅|**dashboardFiltering.spec.ts**|18(2 skipped)|-|232.9730s|
|✅|**attributeFilterButton.spec.ts**|9|-|33.5110s|
|✅|**pivotTableRemoveAllFromMutipleAggregation.spec.ts**|1|-|8.9570s|
|✅|**pivotTableSizing.spec.ts**|1|-|8.1340s|
|✅|**pivotTableAggregationsMenuAndAddTotal.spec.ts**|5|-|24.2430s|
|✅|**pivotTableSizingReset.spec.ts**|15|-|38.7240s|
|✅|**attributeFilterButtonParentChild.spec.ts**|3|-|19.3900s|
|✅|**dateFilter.spec.ts**|9|-|17.5140s|
|✅|**dashboard.spec.ts**|9|-|66.8260s|
|✅|**parentChildFiltersWithPlaceholders.spec.ts**|3|-|22.0690s|
|✅|**barChartDrilling.spec.ts**|4|-|8.0840s|
|✅|**dashboardKPI.spec.ts**|8|-|175.9620s|
|✅|**pivotTableRemoveOneFromMutipleAggregation.spec.ts**|1|-|9.1070s|
|✅|**attributeFilterButtonWithSelection.spec.ts**|2|-|8.7420s|
|✅|**pivotTableRemoveOneAggregation.spec.ts**|1|-|8.1620s|
|✅|**dashboardDateRelated.spec.ts**|8|-|106.1780s|
|❌|**[🎥 drilling.spec.ts](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-docker/369//artifact/libs/sdk-ui-tests-e2e/cypress/videos/drilling.spec.ts.mp4)**|7(2 skipped)|<span style="color: red">1</span>|79.8910s|
|✅|**dashboardDependentFilter.spec.ts**|4|-|79.8840s|
|✅|**headlineChartDrilling.spec.ts**|2|-|4.5030s|

You can find full report [here](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-docker/369/).

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher/66/ : FAILURE in 33m 03s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/11223/ : SUCCESS in 11s

**Comment by MilanLempera** [comment]
extended test - tiger-cypress - isolated

**Comment by yenkins** [comment]
# ✨ Cypress test results ✨

## 😢 Some tests are failing. Consider fixing them.

|Status|Name|Tests|Failures|Time|
|:--:|:--|:--:|:--:|--:|
|❌|**[🎥 dashboardTiger.spec.ts](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-docker/18//artifact/libs/sdk-ui-tests-e2e/cypress/videos/dashboardTiger.spec.ts.mp4)**|9(4 skipped)|<span style="color: red">5</span>|154.2490s|

You can find full report [here](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-docker/18/).

**Comment by MilanLempera** [comment]
extended test - tiger-cypress - record

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher/68/ : FAILURE in 52s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/11225/ : SUCCESS in 11s

**Comment by MilanLempera** [comment]
extended test - tiger-cypress - record dashboardTiger

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher/69/ : FAILURE in 45s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/11226/ : SUCCESS in 11s

**Comment by tuan-tran-thanh** [comment]
extended test - tiger-cypress - record

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher/70/ : FAILURE in 50s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/11227/ : SUCCESS in 12s

**Comment by MilanLempera** [comment]
extended test - cypress - isolated

**Comment by yenkins** [comment]
# ✨ Cypress test results ✨

## 😢 Some tests are failing. Consider fixing them.

|Status|Name|Tests|Failures|Time|
|:--:|:--|:--:|:--:|--:|
|✅|**parentChildFiltersWithPlaceholders.spec.ts**|3|-|16.9440s|
|✅|**dateFilter.spec.ts**|9|-|13.9520s|
|✅|**attributeFilterButtonParentChild.spec.ts**|3|-|13.9700s|
|✅|**headlineChartDrilling.spec.ts**|2|-|3.7820s|
|✅|**pivotTableSizing.spec.ts**|1|-|4.7770s|
|✅|**attributeFilterButtonWithSelection.spec.ts**|2|-|6.8460s|
|✅|**dashboardDnD.spec.ts**|7|-|60.1390s|
|✅|**dashboardFiltering.spec.ts**|18(2 skipped)|-|148.2070s|
|✅|**attributeFilterButton.spec.ts**|9|-|30.1520s|
|✅|**pivotTableSizingReset.spec.ts**|15|-|38.5070s|
|✅|**dashboard.spec.ts**|9|-|58.7130s|
|✅|**dashboardDependentFilter.spec.ts**|4|-|47.7490s|
|✅|**barChartDrilling.spec.ts**|4|-|6.4530s|
|✅|**pivotTableAggregationsMenuAndAddTotal.spec.ts**|5|-|19.7690s|
|✅|**pivotTableRemoveOneFromMutipleAggregation.spec.ts**|1|-|6.9070s|
|✅|**dashboardDateRelated.spec.ts**|8|-|67.4070s|
|✅|**pivotTableRemoveAllFromMutipleAggregation.spec.ts**|1|-|6.8380s|
|❌|**[🎥 drilling.spec.ts](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-docker/371//artifact/libs/sdk-ui-tests-e2e/cypress/videos/drilling.spec.ts.mp4)**|7(2 skipped)|<span style="color: red">1</span>|68.6770s|
|✅|**pivotTableRemoveOneAggregation.spec.ts**|1|-|6.8410s|
|✅|**dashboardKPI.spec.ts**|8|-|142.2220s|

You can find full report [here](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-docker/371/).

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher/75/ : FAILURE in 24m 35s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/11236/ : SUCCESS in 6s

**Comment by MilanLempera** [comment]
extended test - tiger-cypress - record

**Comment by yenkins** [comment]
**Cypress recording succeeded** :tada:

Here is your new recording: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-record-zuul-docker/19/artifact/*zip*/archive.zip

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher/76/ : FAILURE in 7m 42s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/11237/ : SUCCESS in 6s

**Comment by MilanLempera** [comment]
extended test - cypress - isolated

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/11238/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/13338/ : SUCCESS in 15m 09s

**Comment by yenkins** [comment]
# ✨ Cypress test results ✨

## 😢 Some tests are failing. Consider fixing them.

|Status|Name|Tests|Failures|Time|
|:--:|:--|:--:|:--:|--:|
|✅|**dashboardFiltering.spec.ts**|18(2 skipped)|-|156.1010s|
|✅|**dashboardDateRelated.spec.ts**|8|-|71.8160s|
|✅|**dashboardDependentFilter.spec.ts**|4|-|57.6280s|
|✅|**dashboardDnD.spec.ts**|7|-|68.0220s|
|✅|**pivotTableRemoveAllFromMutipleAggregation.spec.ts**|1|-|8.4130s|
|✅|**attributeFilterButtonWithSelection.spec.ts**|2|-|7.1220s|
|✅|**pivotTableRemoveOneFromMutipleAggregation.spec.ts**|1|-|8.8930s|
|✅|**pivotTableSizingReset.spec.ts**|15|-|44.4870s|
|✅|**pivotTableRemoveOneAggregation.spec.ts**|1|-|8.5690s|
|✅|**pivotTableAggregationsMenuAndAddTotal.spec.ts**|5|-|23.0760s|
|✅|**pivotTableSizing.spec.ts**|1|-|6.4850s|
|✅|**headlineChartDrilling.spec.ts**|2|-|5.0010s|
|✅|**barChartDrilling.spec.ts**|4|-|8.4630s|
|✅|**attributeFilterButton.spec.ts**|9|-|30.8690s|
|✅|**attributeFilterButtonParentChild.spec.ts**|3|-|14.5740s|
|✅|**dateFilter.spec.ts**|9|-|18.0730s|
|❌|**[🎥 drilling.spec.ts](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-docker/372//artifact/libs/sdk-ui-tests-e2e/cypress/videos/drilling.spec.ts.mp4)**|7(2 skipped)|<span style="color: red">1</span>|70.6530s|
|✅|**dashboard.spec.ts**|9|-|67.3330s|
|✅|**parentChildFiltersWithPlaceholders.spec.ts**|3|-|20.1590s|
|✅|**dashboardKPI.spec.ts**|8|-|143.9910s|

You can find full report [here](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-docker/372/).

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher/77/ : FAILURE in 26m 05s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/11239/ : SUCCESS in 11s

**Comment by MilanLempera** [comment]
extended test - tiger-cypress - record

**Comment by yenkins** [comment]
**Cypress recording succeeded** :tada:

Here is your new recording: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-record-zuul-docker/21/artifact/*zip*/archive.zip

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher/80/ : FAILURE in 7m 15s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/11244/ : SUCCESS in 6s

**Comment by MilanLempera** [comment]
extended test - tiger-cypress - isolated

**Comment by yenkins** [comment]
# ✨ Cypress test results ✨

## 🎉 Great news! Tests were successful.

|Status|Name|Tests|Failures|Time|
|:--:|:--|:--:|:--:|--:|
|✅|**dashboardTiger.spec.ts**|9(4 skipped)|-|25.0500s|

You can find full report [here](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-docker/22/).

**Comment by yenkins** [comment]
# ✨ Cypress test results ✨

## 🎉 Great news! Tests were successful.

|Status|Name|Tests|Failures|Time|
|:--:|:--|:--:|:--:|--:|
|✅|**dashboardTiger.spec.ts**|9(4 skipped)|-|25.0500s|

You can find full report [here](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-docker/22/).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher/82/ : SUCCESS in 6m 26s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/11247/ : SUCCESS in 12s

**Comment by MilanLempera** [comment]
extended test - tiger-cypress - record

**Comment by yenkins** [comment]
**Cypress recording succeeded** :tada:

Here is your new recording: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-record-zuul-docker/23/artifact/*zip*/archive.zip

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher/84/ : FAILURE in 7m 18s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/11251/ : SUCCESS in 6s

**Comment by MilanLempera** [comment]
extended test - cypress - record

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher/85/ : FAILURE in 59s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/11253/ : SUCCESS in 7s

**Comment by MilanLempera** [comment]
extended test - cypress - isolated

**Comment by gdgate** [comment]
Build in pipeline check-extended aborted.

**Comment by MilanLempera** [comment]
extended test - cypress - isolated

**Comment by yenkins** [comment]
# ✨ Cypress test results ✨

## 🎉 Great news! Tests were successful.

|Status|Name|Tests|Failures|Time|
|:--:|:--|:--:|:--:|--:|
|✅|**dashboardFiltering.spec.ts**|18(2 skipped)|-|225.8460s|
|✅|**attributeFilterButton.spec.ts**|9|-|51.6410s|
|✅|**drilling.spec.ts**|7(2 skipped)|-|71.3750s|
|✅|**dashboard.spec.ts**|9|-|77.6090s|
|✅|**pivotTableSizingReset.spec.ts**|15|-|68.5960s|
|✅|**pivotTableSizing.spec.ts**|1|-|11.7420s|
|✅|**headlineChartDrilling.spec.ts**|2|-|9.2300s|
|✅|**dashboardDependentFilter.spec.ts**|4|-|65.7240s|
|✅|**attributeFilterButtonWithSelection.spec.ts**|2|-|14.1240s|
|✅|**pivotTableRemoveAllFromMutipleAggregation.spec.ts**|1|-|14.3880s|
|✅|**barChartDrilling.spec.ts**|4|-|13.3080s|
|✅|**parentChildFiltersWithPlaceholders.spec.ts**|3|-|28.0550s|
|✅|**dashboardKPI.spec.ts**|8|-|268.5640s|
|✅|**pivotTableRemoveOneFromMutipleAggregation.spec.ts**|1|-|16.3250s|
|✅|**dashboardDateRelated.spec.ts**|8|-|94.0010s|
|✅|**pivotTableAggregationsMenuAndAddTotal.spec.ts**|5|-|40.7620s|
|✅|**attributeFilterButtonParentChild.spec.ts**|3|-|24.8720s|
|✅|**dashboardDnD.spec.ts**|7|-|80.0590s|
|✅|**pivotTableRemoveOneAggregation.spec.ts**|1|-|15.6240s|
|✅|**dateFilter.spec.ts**|9|-|34.9340s|

You can find full report [here](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-docker/374/).

**Comment by yenkins** [comment]
# ✨ Cypress test results ✨

## 😢 Some tests are failing. Consider fixing them.

|Status|Name|Tests|Failures|Time|
|:--:|:--|:--:|:--:|--:|
|✅|**dashboardKPI.spec.ts**|8|-|142.8010s|
|✅|**dashboardDependentFilter.spec.ts**|4|-|48.4310s|
|❌|**[🎥 drilling.spec.ts](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-docker/375//artifact/libs/sdk-ui-tests-e2e/cypress/videos/drilling.spec.ts.mp4)**|7(2 skipped)|<span style="color: red">1</span>|68.3260s|
|✅|**attributeFilterButtonWithSelection.spec.ts**|2|-|6.9720s|
|✅|**pivotTableSizingReset.spec.ts**|15|-|38.3530s|
|✅|**headlineChartDrilling.spec.ts**|2|-|3.8750s|
|✅|**pivotTableAggregationsMenuAndAddTotal.spec.ts**|5|-|19.7540s|
|✅|**barChartDrilling.spec.ts**|4|-|6.4440s|
|✅|**pivotTableRemoveOneFromMutipleAggregation.spec.ts**|1|-|7.1500s|
|✅|**pivotTableRemoveOneAggregation.spec.ts**|1|-|6.8870s|
|✅|**dashboardFiltering.spec.ts**|18(2 skipped)|-|150.0920s|
|✅|**pivotTableRemoveAllFromMutipleAggregation.spec.ts**|1|-|6.9090s|
|✅|**pivotTableSizing.spec.ts**|1|-|4.7470s|
|✅|**attributeFilterButtonParentChild.spec.ts**|3|-|13.9030s|
|✅|**attributeFilterButton.spec.ts**|9|-|29.9830s|
|✅|**dashboardDnD.spec.ts**|7|-|58.7580s|
|✅|**dashboard.spec.ts**|9|-|59.2800s|
|✅|**parentChildFiltersWithPlaceholders.spec.ts**|3|-|17.1470s|
|✅|**dateFilter.spec.ts**|9|-|13.9330s|
|✅|**dashboardDateRelated.spec.ts**|8|-|68.3170s|

You can find full report [here](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-docker/375/).

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher/90/ : FAILURE in 23m 39s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/11261/ : SUCCESS in 8s

**Comment by MilanLempera** [comment]
extended test - cypress - isolated

**Review by hackerstanislav — APPROVED** [review]

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: libs/sdk-ui-dashboard/src/presentation/layout/DefaultDashboardLayoutRenderer/DashboardLayoutGridRowEdit.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+                      section,
+                      items,
+                      renderMode,
+                  })
+                : extendedRows}
+        </Row>
+    );
+}
+
+function splice(arr: any[], start: number, deleteCount: number, ...addItem: any[]) {
```
Comment (kandl) [inline libs/sdk-ui-dashboard/src/presentation/layout/DefaultDashboardLayoutRenderer/DashboardLayoutGridRowEdit.tsx:85]: Should this rather be "insert"? (as we are not using it for any removal)

**Comment by yenkins** [comment]
# ✨ Cypress test results ✨

## 🎉 Great news! Tests were successful.

|Status|Name|Tests|Failures|Time|
|:--:|:--|:--:|:--:|--:|
|✅|**pivotTableRemoveOneFromMutipleAggregation.spec.ts**|1|-|7.0620s|
|✅|**pivotTableSizing.spec.ts**|1|-|4.7390s|
|✅|**dashboardKPI.spec.ts**|8|-|145.6570s|
|✅|**dashboardFiltering.spec.ts**|18(2 skipped)|-|149.1440s|
|✅|**attributeFilterButtonWithSelection.spec.ts**|2|-|6.8570s|
|✅|**dateFilter.spec.ts**|9|-|13.9600s|
|✅|**dashboardDnD.spec.ts**|7|-|60.2130s|
|✅|**attributeFilterButtonParentChild.spec.ts**|3|-|14.1150s|
|✅|**drilling.spec.ts**|7(2 skipped)|-|40.3860s|
|✅|**pivotTableSizingReset.spec.ts**|15|-|38.7540s|
|✅|**headlineChartDrilling.spec.ts**|2|-|3.7890s|
|✅|**dashboard.spec.ts**|9|-|58.2560s|
|✅|**pivotTableAggregationsMenuAndAddTotal.spec.ts**|5|-|19.8280s|
|✅|**pivotTableRemoveOneAggregation.spec.ts**|1|-|6.7990s|
|✅|**dashboardDateRelated.spec.ts**|8|-|66.0050s|
|✅|**attributeFilterButton.spec.ts**|9|-|29.8950s|
|✅|**barChartDrilling.spec.ts**|4|-|6.4750s|
|✅|**pivotTableRemoveAllFromMutipleAggregation.spec.ts**|1|-|6.7530s|
|✅|**parentChildFiltersWithPlaceholders.spec.ts**|3|-|17.0580s|
|✅|**dashboardDependentFilter.spec.ts**|4|-|48.6880s|

You can find full report [here](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-docker/376/).

**Comment by yenkins** [comment]
# ✨ Cypress test results ✨

## 🎉 Great news! Tests were successful.

|Status|Name|Tests|Failures|Time|
|:--:|:--|:--:|:--:|--:|
|✅|**pivotTableRemoveOneFromMutipleAggregation.spec.ts**|1|-|7.0620s|
|✅|**pivotTableSizing.spec.ts**|1|-|4.7390s|
|✅|**dashboardKPI.spec.ts**|8|-|145.6570s|
|✅|**dashboardFiltering.spec.ts**|18(2 skipped)|-|149.1440s|
|✅|**attributeFilterButtonWithSelection.spec.ts**|2|-|6.8570s|
|✅|**dateFilter.spec.ts**|9|-|13.9600s|
|✅|**dashboardDnD.spec.ts**|7|-|60.2130s|
|✅|**attributeFilterButtonParentChild.spec.ts**|3|-|14.1150s|
|✅|**drilling.spec.ts**|7(2 skipped)|-|40.3860s|
|✅|**pivotTableSizingReset.spec.ts**|15|-|38.7540s|
|✅|**headlineChartDrilling.spec.ts**|2|-|3.7890s|
|✅|**dashboard.spec.ts**|9|-|58.2560s|
|✅|**pivotTableAggregationsMenuAndAddTotal.spec.ts**|5|-|19.8280s|
|✅|**pivotTableRemoveOneAggregation.spec.ts**|1|-|6.7990s|
|✅|**dashboardDateRelated.spec.ts**|8|-|66.0050s|
|✅|**attributeFilterButton.spec.ts**|9|-|29.8950s|
|✅|**barChartDrilling.spec.ts**|4|-|6.4750s|
|✅|**pivotTableRemoveAllFromMutipleAggregation.spec.ts**|1|-|6.7530s|
|✅|**parentChildFiltersWithPlaceholders.spec.ts**|3|-|17.0580s|
|✅|**dashboardDependentFilter.spec.ts**|4|-|48.6880s|

You can find full report [here](https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-docker/376/).

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher/92/ : SUCCESS in 21m 42s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/11263/ : SUCCESS in 9s

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR3420

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/285/ : FAILURE in 3m 30s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/3552/ : SUCCESS in 21s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/2400/ : SUCCESS in 19m 14s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/13349/ : SUCCESS in 12m 33s

**Pull request merged**